### DISCUSSION TITLE
Conditions attribute as property format

### DISCUSSION THREAD

**Discussion by leroyloren** [zachowj/node-red-contrib-home-assistant-websocket#667] [Q&A]
I have this attribute available for the sensor:
**a92ee200-5501-11e4-916c-0800200c9a66 7266.30133**

Can this format be loaded as a property in the trigger state?
![obrazek](https://user-images.githubusercontent.com/57643470/192601947-bbd7ef2b-c176-4246-8e3b-6c758e2bf6a5.png)

**Comment by zachowj** [comment]
not sure if it will work but you could try

```
new_state.attributes['a92ee200-5501-11e4-916c-0800200c9a66 7266.30133']
```

**Comment by leroyloren** [comment]
Unfortunately, it doesn't work either.

Despite the function, it's fine
`msg.data.event.new_state.attributes["b5b182c7-eab1-4988-aa99-b5c1517008d9 1.55963"];`

But I would like to trigger directly

**Reply by zachowj** [reply]
try 
```
new_state.attributes.a92ee200-5501-11e4-916c-0800200c9a66 7266\\.30133
```

**Comment by leroyloren** [comment]
Doesn't work either 😔

**Comment by zachowj** [comment]
The above worked in my test, let me create it in NR and test

```
const selectn = require("selectn");

const obj = {
  new_state: {
    attributes: {
      "a92ee200-5501-11e4-916c-0800200c9a66 7266.30133": 123,
    },
  },
};

console.log(
  selectn(
    "new_state.attributes.a92ee200-5501-11e4-916c-0800200c9a66 7266\\.30133",
    obj
  )
);

```

**Reply by zachowj** [reply]
Interesting, when I try and create a sensor with that attribute HA sends back a modified name. It converts the space to an underscore and removes the `.`, If you use a current state node and dump the entity using a debug node what does it show for attributes?

```
attributes: {
      'a92ee200-5501-11e4-916c-0800200c9a66_726630133': '2',
      friendly_name: 'nodered 48291a57667670b4'
    },
```

**Comment by leroyloren** [comment]
Now I don't understand, should I enter the code somewhere?

this is what the sensor looks like
![obrazek](https://user-images.githubusercontent.com/57643470/192607801-0245b64c-77e6-473d-8af3-aa99584b9a4f.png)

Debug output
`{"topic":"sensor.pixel_6_beacon_monitor","payload":"Monitoring","data":{"event_type":"state_changed","entity_id":"sensor.pixel_6_beacon_monitor","event":{"entity_id":"sensor.pixel_6_beacon_monitor","old_state":{"entity_id":"sensor.pixel_6_beacon_monitor","state":"Monitoring","attributes":{"icon":"mdi:bluetooth","friendly_name":"Pixel 6 Beacon Monitor"},"last_changed":"2022-09-27T18:24:40.149026+00:00","last_updated":"2022-09-27T18:32:54.038848+00:00","context":{"id":"01GE03JB0PN9NRB9KX3A1FDQG9","parent_id":null,"user_id":null},"original_state":"Monitoring"},"new_state":{"entity_id":"sensor.pixel_6_beacon_monitor","state":"Monitoring","attributes":{"a92ee200-5501-11e4-916c-0800200c9a66 7266.30133":1.64,"icon":"mdi:bluetooth","friendly_name":"Pixel 6 Beacon Monitor"},"last_changed":"2022-09-27T18:24:40.149026+00:00","last_updated":"2022-09-27T18:32:57.200133+00:00","context":{"id":"01GE03JE3GQF937F0XMJRBSBAY","parent_id":null,"user_id":null},"original_state":"Monitoring","timeSinceChangedMs":497063}},"origin":"LOCAL","time_fired":"2022-09-27T18:32:57.200133+00:00","context":{"id":"01GE03JE3GQF937F0XMJRBSBAY","parent_id":null,"user_id":null}},"_msgid":"0f3c3adde4934ef2","beacon":1.64}`

**Reply by zachowj** [reply]
If escaping the period with a double forward slash doesn't work. I don't think the current implementation will work then. You might have to do it in a function node.

**Comment by leroyloren** [comment]
@zachowj What do you think?

https://github.com/home-assistant/android/pull/2941
